### PR TITLE
#159951825 Add Pagination support for articles

### PR DESCRIPTION
**Pull request by ssewilliam**
### What does this PR do?
- Adds Pagination to articles

### How should this be manually tested?
- browse to articles and scroll infinitely as articles are paginated

### Any background context you want to provide?
- I used a package react-infinite-scroll-component
- I also implemented Redux to sore the paginated articles

### What are the relevant pivotal tracker stories?
- [#159951825](https://www.pivotaltracker.com/story/show/159951825)

### coverage screenshot
screen shot 2018-09-17 at 20 11 24

### CODE REVIEW

**Review by lytes20 — APPROVED** [review]

**Review by phillipseryazi — APPROVED** [review]

**Review by abulojoshua1 — APPROVED** [review]

File: src/CSS/Article.css [hunk 1]
```diff
@@ -53,4 +54,44 @@
 label{
     background: none !important;
     margin-left: -10px;
+    padding: 6px 3px;
```
Comment (RutaleIvanPaul) [inline src/CSS/Article.css]: Can you use a different styling element? I feel there's going to be unforeseen style problems when you use this one.

Comment (RutaleIvanPaul) [reply src/CSS/Article.css]: @ssewilliam

**Comment by AGMETEOR** [comment]
@ssewilliam like functionality is not working. Find a way to pass article slug as props to the Like component ;)

**Comment by ssewilliam** [comment]
@AGMETEOR, I am following up on this as well, and it should be working shortly.

**Review by AGMETEOR — APPROVED** [review]

**Pull request merged**